datalad/interface/clean.py

```diff
@@ -91,13 +94,17 @@ def __call__(dataset=None, what=None, recursive=False, recursion_limit=None):
                 continue
             d = ap['path']
             gitdir = GitRepo.get_git_dir(d)
+            DIRS_PLURAL = ("directory", "directories")
+            FILES_PLURAL = ("file", "files")
```

Comment:
Why uppercase?

Reply:
Old fashion way to make constants

Reply:
OK, just curious.  Given that these are just used locally (and immediately) within `__call__`, it reads oddly to me, but it of course doesn't matter either way.